parlai/crowdsourcing/utils/tests.py

```diff
@@ -215,7 +273,10 @@ def _get_agent_state(self, task_data: Dict[str, Any]):
         """
 
         # Set up the mock human agent
-        agent_id = self._register_mock_agents(num_agents=1)[0]
+        if self.config.mephisto.blueprint.get("onboarding_qualification", None):
```

Comment:
is there a better way to check if we should test onboarding? Would it be better to pass an onboarding bool variable?

Reply:
At the moment this is how we specify onboarding in Mephisto:
https://github.com/facebookresearch/Mephisto/blob/main/mephisto/abstractions/blueprints/mixins/onboarding_required.py#L103-L107
So I think this is really the only way to check that a config is trying to use onboarding.